examples/tracing-http-propagator/src/server.rs

```diff
@@ -66,17 +77,20 @@ async fn router(
 ) -> Result<Response<BoxBody<Bytes, hyper::Error>>, Infallible> {
     // Extract the context from the incoming request headers
     let parent_cx = extract_context_from_request(&req);
+    for bag in parent_cx.baggage() {
+        println!("Baggage: {:?} = {:?}", bag.0, bag.1);
```

Comment:
why not use the logger?

Reply:
this was a leftover from my local testing, will remove!